github/authorizations.go

```diff
@@ -85,6 +85,20 @@ func (a AuthorizationApp) String() string {
 	return Stringify(a)
 }
 
+// OAuthGrant represents an OAuth application that has been granted access to an account.
+type Grant struct {
+	ID        *int              `json:"id,omitempty"`
+	URL       *string           `json:"url,omitempty"`
+	App       *AuthorizationApp `json:"app,omitempty"`
+	CreatedAt *Timestamp        `json:"created_at, omitempty"`
+	UpdatedAt *Timestamp        `json:"updated_at,omitempty"`
+	Scopes    []string          `json:"scopes,omitempty"`
+}
+
+func (a Grant) String() string {
```

Comment:
Nitpick, but the receiver name `a` doesn't have anything in common with `Grant`. How about naming it `g` instead?